root/edit/details/AddWork.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * @flow
+ * Copyright (C) 2020 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import commaOnlyList from '../../static/scripts/common/i18n/commaOnlyList';
+
+
+ type AddWorkEditProps = {
+   ...EditT,
+   +display_data: {
+     +attributes?: AttributeT,
+     +comment?: string,
+     +iswc?: string,
+     +language?: LanguageT,
+     +languages: $ReadOnlyArray<string>,
+     +name: string,
+     +type?: WorkTypeT,
```

Comment:
That's always included, but can be `null` if the type is unset or can't be loaded. See `build_display_data`.
```suggestion
     +type: WorkTypeT | null,
```